### PR TITLE
[ISSUE #7229] Optimize the constants to the identify package for the client module. Part 3

### PR DESCRIPTION
**Pull request by onewe**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

sub pr #7229

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by KomachiSion** [comment]
It seems some problem in CI, I tried many times and CI has been cancelled for each timed. I think you should review which changes make this problem.

I will try last time again, If can't pass and be cancelled again, we will reject this PR before fixed.

**Comment by onewe** [comment]
@KomachiSion It's strange. I don't reproduce this problem at local.😮‍💨

File: client/src/main/java/com/alibaba/nacos/client/identify/CredentialWatcher.java [hunk 1]
```diff
@@ -187,26 +209,26 @@ private void loadCredential(boolean init) {
                 SPAS_LOGGER.info("[{}] Load credential file {}", appName, propertyPath);
             }
             
-            if (!IdentifyConstants.DOCKER_CREDENTIAL_PATH.equals(propertyPath)) {
-                if (properties.containsKey(IdentifyConstants.ACCESS_KEY)) {
-                    accessKey = properties.getProperty(IdentifyConstants.ACCESS_KEY);
+            if (!DOCKER_CREDENTIAL_PATH.equals(propertyPath)) {
+                if (properties.containsKey(ACCESS_KEY)) {
```
Comment (KomachiSion) [inline client/src/main/java/com/alibaba/nacos/client/identify/CredentialWatcher.java:213]: For some constants which may get from properties. They should be still use IdentityConstants.

The reason is users may use the IdentityConstants to init properties key before they call `Factory`

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: client/src/main/java/com/alibaba/nacos/client/identify/CredentialWatcher.java (hunk 1 above)
Comment (onewe) [reply client/src/main/java/com/alibaba/nacos/client/identify/CredentialWatcher.java:213]: Alright.

**Review by realJackSun — DISMISSED** [review]

**Pull request closed without merging**